standard/info/wikis/geoguessr/info.lua

```diff
@@ -0,0 +1,37 @@
+---
+-- @Liquipedia
+-- wiki=geoguessr
+-- page=Module:Info
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+return {
+	startYear = 2013,
+	wikiName = 'geoguessr',
+	name = 'GeoGuessr',
+	defaultGame = 'geoguessr',
+	games = {
+		geoguessr = {
+			abbreviation = 'GeoGuessr',
+			name = 'GeoGuessr',
+			link = 'GeoGuessr',
+			logo = {
+				darkMode = 'GeoGuessr allmode.png',
+				lightMode = 'GeoGuessr allmode.png',
+			},
+			defaultTeamLogo = {
+				darkMode = 'GeoGuessr allmode.png',
+				lightMode = 'GeoGuessr allmode.png',
+			},
+		},
+	},
+	config = {
+		squads = {
+			hasPosition = false,
+			hasSpecialTeam = false,
+			allowManual = true,
+		},
+	},
+	match2 = 1,
```

Comment:
either 0 or 2
- pov 1: match2 is not set up yet, hence 0
- pov 2: there are no old matches present, hence 2

maybe @Rathoz can tell us what he thinks is most applicable
in any case 1 seems wrong

Reply:
I've updated it to 0 for now 
(currently: Match2 is temporarily setup just so tournaments work, will be on PR soon so I originally put 1 to it)